include/highfive/H5PropertyList.hpp

```diff
@@ -147,15 +144,15 @@ class Caching {
     /// details.
     Caching(const size_t numSlots,
             const size_t cacheSize,
-            const double w0 = H5D_CHUNK_CACHE_W0_DEFAULT)
+            const double w0 = static_cast<double>(H5D_CHUNK_CACHE_W0_DEFAULT))
```

Comment:
IMHO one should always use proper type conversion mechanisms provided by C++ instead of c-style cast that can convert carrots to apples without complaining.

Reply:
I agree, my thought was that eventually we could use the short syntax only for primitive types. But it's fine for me.